draft-ietf-moq-transport.md

```diff
@@ -442,6 +442,27 @@ constrain the information in these fields, for example by restricting them to
 UTF-8. Any specification that does needs to specify the canonicalization into
 the bytes in the Track Namespace or Track Name such that exact comparison works.
 
+### Subscription Identifiers {#subscription-id}
+
+To optimize wire efficiency, some MoQT messages refer to a track by a numeric
+identifier, rather than the Full Track Name. When a subscriber makes a SUBSCRIBE
+or FETCH request for a Track, it associates a Request ID with the request.  The
+Request ID (RSID) can be used as a Subscription ID by shifting it left by one,
+eg: `Request Subscription ID = Request ID << 1`.
+
+MoQT also allows the publisher to choose an identifier for each subscription,
+called the Publisher-Chosen Subscription ID (PSID). PSIDs are odd and have a
+maximum value of 2^62-1. The same identifier MUST NOT be assigned to different
+Tracks in the same session.  The publisher is not required to specify a PSID.
+
+The term Subscription ID refers to either a Request Subscription ID or
+Publisher-Chosen Subscription ID, determined by the least significant bit of
+the identifier.  Subgroups, Datagrams as well as control messages for
+established subscriptions such as SUBSCRIBE_UPDATE, UNSUBSCRIBE and
+SUBSCRIBE_DONE use Subscription IDs.  The sender chooses the identifier type
+based on its preference.
```

Comment:
I will remove the last sentence as it gets more clarified in the relay section OR i would also add reference to the relay section here